### PR TITLE
Exceptions

### PR DESCRIPTION
**Pull request by bikashkanungo**

### CODE REVIEW

**Comment by dsambit** [comment]
@bikashkanungo please run from the top directory

> ./devHelpers/indentationStandard/indent-all

**Review by dsambit — APPROVED** [review]

File: src/utils/Exceptions.h [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    throwException(bool condition, std::string msg = "");
+
+    template <class T>
+    void
+    throwException(bool condition, std::string msg = "");
+
+  } // namespace utils
+} // namespace dftefe
+#include "Exceptions.t.cpp"
+#endif // dftefeAssert_h
```
Comment (dsambit) [inline src/utils/Exceptions.h:107]: minor type: comment should be "dftefeExceptions_h"

Comment (bikashkanungo) [reply src/utils/Exceptions.h]: Done

**Comment by dsambit** [comment]
@bikashkanungo you can go ahead and merge this PR

**Pull request merged**